### ISSUE TITLE
Trained DETR paramters on v-coco

### ISSUE THREAD

**Issue by FeiX-OvO** [hitachi-rd-cv/qpic#16]
Hi, thanks for your work! I wish to do the training for v-coco, do I need to first train detr with the un-overlapped data and then QPIC? Is it possible for you to provide the trained parameters for DETR? Thanks.

**Comment by ASMIftekhar** [comment]
Hello 
I am also interested to get the pre-trained DETR model. I really appreciate your efforts to make the repo simple and I can totally understand your reservation on releasing something that is not the key point in your model. However, training on COCO takes a huge amount of time. Therefore, this would be a huge favor to hoi detection community.

**Comment by JacobYuan7** [comment]
But if we want to do the pretraining ourselves, what should we do?  How can we find the V-COCO test set images that are contained in the COCO train2017 set? I notice that the image names of the V-COCO test set are all in the style like 'COCO_val2014_xxxxxxxxxxxx.jpg'.

**Comment by tamtamz** [comment]
'xxxxxxxxxxxx' part of 'COCO_val2014_xxxxxxxxxxxx.jpg' is an image id. The ids are identical in 2014 set and 2017 set.
